PennMobile/General/Protocols.swift

```diff
@@ -145,3 +146,41 @@ extension LocationPermissionRequestable {
         }
     }
 }
+
+protocol LocalAuthentication {
```

Comment:
Let's change this protocol name so that it's not the same as the Apple Package. Maybe `LocallyAuthenticatable` or something else that reads nicely.